### PR TITLE
Remove last mojom interface

### PR DESCRIPTION
**Pull request by abarth**
We no longer use mojom to transport messages. We still use the Mojo EDK
to spin the event loop, however.

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde @jason-simmons

**Comment by jason-simmons** [comment]
LGTM

File: runtime/dart_init.cc [hunk 1]
```diff
@@ -608,12 +605,12 @@ void InitDartVM() {
 
 #if defined(OS_FUCHSIA) && defined(NDEBUG)
   if (false) {
-    // Do not enable checked mode for Fuchsia release builds
-    // TODO(mikejurka): remove this once precompiled code is working on Fuchsia
+// Do not enable checked mode for Fuchsia release builds
```
Comment (Hixie) [inline runtime/dart_init.cc]: broken indentation?

**Comment by Hixie** [comment]
RSLGTM

File: runtime/dart_init.cc (hunk 1 above)
Comment (abarth) [inline runtime/dart_init.cc]: Fixed.

**Pull request merged**